ets-demo/etstool.py

```diff
@@ -311,6 +336,62 @@ def flake8(runtime, toolkit, environment):
     execute(commands, parameters)
 
 
+@cli.command(name="generate-bundles")
+def generate_bundles():
+    """ Generate application bundles for each platform to be uploaded to
+    enthought/free repository
+    """
+
+    for platform in PLATFORMS:
+        bundle_dir = os.path.join("bundle", platform)
+        if not os.path.exists(bundle_dir):
+            os.makedirs(bundle_dir)
+
+        bundle_file = os.path.join(
+            bundle_dir,
+            f"etsdemo_{platform.replace('-', '_')}-{APP_BUNDLE_VERSION}.bundle"
+        )
+    
+        click.echo("Generating bundle {}".format(bundle_file))
+
+        edm_command = [
+            "edm",
+            "bundle",
+            "generate",
+            "--bundle-format",
+            "2.0",
+            "--platform",
+            platform,
+            "--version=3.6",
+            "--output-file",
+            bundle_file,
+        ] + sorted(full_app_dependencies)
+        try:
+            subprocess.check_call(edm_command)
+        except subprocess.CalledProcessError:
+            click.echo("Failed to generate bundle {}".format(bundle_file))
+
+
+@cli.command(name="upload-bundles")
+@click.option('--repository', default=DEFAULT_REPOSITORY)
```

Comment:
More machinery will be needed to use non `enthought/free` repository.  "If the bundle is uploaded to any repository other than enthought/free, the edm yaml config file needs to be updated with a new bundle_repositories entry, pointing to the repository containing the bundle file."

Reply:
This won't be a problem as we only intend to upload to `enthought/free` - and all packages we rely on will be available via `enthought/free`